.github/workflows/release.yml

```
@@ -0,0 +1,202 @@
+name: Create NuGet and GitHub Release
+
+on:
+  pull_request:
+    types:
+      - closed
+  workflow_dispatch:
+
+
+permissions:
+  contents: write
+
+### TODO: Replace instances of './.github/workflows/' w/ `auth0/dx-sdk-actions/workflows` and append `@latest` after the common `dx-sdk-actions` repo is made public.
+### TODO: Also remove `get-prerelease`, `get-release-notes`, `get-version`, `release-create`, and `tag-exists` actions from this repo's .github/actions folder once the repo is public.
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    environment: 'release'
+    if: github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && github.event.pull_request.merged && startsWith(github.event.pull_request.head.ref, 'release/'))
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+
+      - name: Install .NET
+        uses: actions/setup-dotnet@v3
+
+      - name: Install .NET Android and iOS workload
+        run: dotnet workload install android ios maui
+
+      - name: Setup NuGet
+        uses: nuget/setup-nuget@v1
+
+      - name: Restore NuGet
+        run: nuget restore Auth0.OidcClient.All.sln
+
+      - name: Setup MSBuild
+        uses: microsoft/setup-msbuild@v1.1
+
+      - name: Build
+        run: msbuild Auth0.OidcClient.All.sln -t:rebuild -verbosity:diag -property:Configuration=Release
+
+      - uses: actions/upload-artifact@v4
```

Is it worth downgrading this to v3 as there seems to be issues amongst the latest *-artifact action releases
```suggestion
      - uses: actions/upload-artifact@v3
```